### ISSUE TITLE
Remove the ability to login as a new user without manually clearing the app data

### ISSUE THREAD

**Issue by cafootitt** [OpenSRP/opensrp-client-chw#1382]
App version 1.0.0 (built 23 Sept) - production apk

Steps to replicate:
- Login as a user
- Logout and try to login as a different user
- The app shows the following message:

![Screenshot_20201002-140939_CFC-RTM - RDC](https://user-images.githubusercontent.com/3488986/94957143-33528500-04e5-11eb-9e58-4d2f731c99fb.jpg)

- If you press Ok, you get an error message and can't log in.
- The unicef team reported that instead of the error message that I got, they were able to log in, but the data from the previous user was still shown.

In any event, the unicef team is requesting that we remove this feature completely for all wcaro flavors, as they do not want another user accessing the same device as another user.

Please apply this change to all WCARO flavors.

**Comment by cafootitt** [comment]
One blocking PR from BA (Allan and Pauline working on this), and aim is to merge that today, so we should be able to get this merged tomorrow.

**Comment by cafootitt** [comment]
Once keycloak branch is merged, we can fix this. Zain can look into this now.

**Comment by cafootitt** [comment]
Ronald to confirm the set up of these two users: demo and demo1
demo is working
demo1 is not logging in, so Ronald will check configuration of this user.

**Comment by cafootitt** [comment]
DRC V1.0.1 - lgtm. We'll update the error message in a future release (new issue to be created).
